charts/cluster-api-runtime-extensions-nutanix/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: trust-manager
```

Comment:
I'm not keen on trust-manager being a requirement for CAREN - shouldn't this be a user deployment configuration rather than a strict requirement? It could conflict with existing trust-manager deployment for example.

Reply:
We can keep this functionality until we implement https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/497

We might be able to then set this behind a boolean gate. If that is the case, i'm not sure what the implication for konvoy would be. How should we template it? 

On the other hand, CAREN is already a highly opinionated and has a ton of dependencies adding another isn't that big of a deal IMO